README.md

```diff
@@ -421,4 +423,4 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Hostelworld](https://www.hostelworld.com/) [:rocket:](https://www.hostelworldgroup.com/careers) | World's leading hostel-focused online booking platform. | `Porto` |
 | [Ryanair](https://www.ryanair.com/pt/pt) [:rocket:](https://careers.ryanair.com/jobs/?ryanair-jobs-department=21233) | Ryanair is an Irish ultra low-cost carrier. | `Lisboa` `Porto`|
 | [TripAdvisor](https://www.tripadvisor.com) [:rocket:](https://careers.tripadvisor.com) | The largest "social travel website" in the world. | `Lisboa` |
-
+| [Vision-Box](https://www.vision-box.com/) [:rocket:](https://visionbox.teamtailor.com/jobs) | End-to-end solutions that combine a comprehensive software platform with seamless and contactless state-of-the-art equipment, making a travel journey feel effortless. | `Lisboa` |
```

Comment:
The description is a very long, could you please shorten it?

Reply:
Changed the description, hopefully is more aligned.